datasets/cats_vs_dogs.py

```diff
@@ -35,14 +35,17 @@ def catsdogs_get_datasets(data, load_train=True, load_test=True):
         is_dir = os.path.isdir(dataset_path)
 
         if not is_dir:
-            print("******************************************")
+            print("\n******************************************")
             print("Please follow instructions below:")
-            print("Download the dataset in the current working directory by visiting this link"
+            print("Download the dataset in the \'ai8x-training\' directory by visiting this link"
```

Comment:
This should be \'ai8x-training/data\'

Reply:
The script will take care of putting the dataset into 'ai8x-training/data' directory and remove the downloaded file from 'ai8x-training' directory.